operations/helm/charts/mimir-distributed/CHANGELOG.md

```diff
@@ -33,6 +33,7 @@ Entries should include a reference to the Pull Request that introduced the chang
 * [ENHANCEMENT] Sets the `appProtocol` value to `tcp` for the `gossip-ring-svc` service template. This allows memberlist to work with istio protocol selection. #5673
 * [ENHANCEMENT] Update the `rollout-operator` subchart to `0.8.0`. #5718
 * [ENHANCEMENT] Make store_gateway podManagementPolicy configurable. #5757
+* [ENHANCEMENT] Set `maxUnavailable` to 0 for most deployments, to ensure they don't become completely unavailable during a rollout. #5924
```

Comment:
nitpick but can you list all the deployments this affects? or all the deployments it doesn't affect?

Reply:
Good idea, added in 1e99303.